### ISSUE TITLE
escape `$` in images page.

### ISSUE THREAD

**Issue by zkamvar** [carpentries/sandpaper#444]
Images with `$` in alt text will cause Mathjax to get excited if there are `$` symbols in there. These should be escaped with `\` before being processed in the images page. 

The relevant place where this text should be escaped is:

https://github.com/carpentries/sandpaper/blob/3d088c7fe81191d665cb8a812aff737e5c1b145f/R/build_images.R#L49-L59

**Comment by zkamvar** [comment]
I do not think this is actually a problem anymore because https://github.com/carpentries/varnish/pull/90 made the math detection not so greedy
